sa/storage-authority.go

```diff
@@ -429,6 +466,33 @@ func (ssa *SQLStorageAuthority) UpdatePendingAuthorization(authz core.Authorizat
 		return
 	}
 
+	var challs []challModel
+	_, err = tx.Select(
+		&challs,
+		"SELECT * FROM challenges WHERE authorizationID = :authID",
```

Comment:
@rolandshoemaker pointed out that GetAuthorization returns the Authorization view (not model), which in turn contains the Challenge view (not model), which doesn't contain the necessary ID field. This will be fixed by https://github.com/letsencrypt/boulder/issues/638, but accepting it as technical debt for now to avoid exploding the scope of this change.